### ISSUE TITLE
Post Formatting Issue

### ISSUE THREAD

**Issue by jmcblane** [michael-lazar/rtv#649]
Not sure when this started, but suddenly, regardless of the editor I use, my comments and posts are not being formatted properly when I send them. I will type something like this in the editor:

```
This is a post

There is an empty line between this because I want a new paragraph.

It should create three paragraphs, and used to do it just fine.
```

But, when I view the comment on reddit, what I see is:

>This is a post There is an empty line between this because I want a new paragraph. It should create three paragraphs, and used to do it just fine.

Thus, everything I submit becomes an ugly wall of text!

I have tested vim, gvim, neovim, nano... ~~About to test mousepad and report back.~~ Same thing in mousepad. Not an editor specific issue.

**Comment by michael-lazar** [comment]
Thanks for diagnosing the issue. The formatting code was recently updated to fix leading whitespace with code blocks. It's likely that a regression was introduced that is stripping empty lines.

https://github.com/michael-lazar/rtv/pull/642

**Comment by michael-lazar** [comment]
I just released bugfix version 1.25.1 to address this issue

**Comment by jmcblane** [comment]
Excellent work! Mighty fast!

:+1:
